aruba.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "cucumber", [">= 2.4", "< 6.0"]
   spec.add_runtime_dependency "rspec-expectations", "~> 3.4"
   spec.add_runtime_dependency "thor", "~> 1.0"
+  spec.add_runtime_dependency "bundler", "~> 2.1"
```

Comment:
Let's allow both bundler 1.x and 2.x since 1.x may be in use on older Rubies. So something like `[">= 1.0", "< 3.0"]`.

Also rubocop wants the dependencies to be sorted alphabetically.